.gitignore

```diff
@@ -38,6 +38,20 @@ crashlytics.properties
 crashlytics-build.properties
 fabric.properties
 
+# Database entities
+backend/com.tngtech.apicenter.backend.connector.database.entity.SpecificationEntity
+
+# Single-deploy additional Gradle directories
+frontend/.gradle
+.gradle/
+
+# Angular build directories
+frontend/build/
+backend/src/main/resources
```

Comment:
I'm afraid this is too broad; sometimes, there are useful files in the folder. Can we just ignore the subfolder or something?